telegram/start_telegram_bot.js

```diff
@@ -191,16 +192,58 @@ module.exports = ({fs, id, limits, lnds, logger, payments, request}, cbk) => {
         });
       }],
 
+      //Get proxy agent
+      getProxyAgent: ['apiKey', 'saveKey', ({}, cbk) => {
+        //Exit early if not using a proxy
+        if(!proxy) {
+          return cbk();
+        }
+
+        const path = join(...[homedir(), home, proxyFile]);
+
+        return fs.getFile(path, (err, res) => {
+        
+          if(!!err) {
+            logger.error({error: err});
+          }
+
+          const proxyData = JSON.parse(res);
```

Comment:
The user might put bad data in the file, should catch that and validate the proxy data is reasonable

Reply:
what sort of validations need to be done? like the right JSON key names are used, like host, port, etc? and also the data type of the values are strings, etc?

Reply:
If you wrap JSON.parse in a try catch it will make sure the JSON they put is valid and then yeah if host/port is required make sure that looks reasonable

Reply:
I added a commit to do the changes. Can't really validate if anything is required, depends on the type of proxy setup a user will use I guess.